src/address.js

```diff
@@ -240,10 +270,11 @@ Returns null if no address is found.
   /**
    * Retrieves the latest account and token balances for the specified address(es).
    *
-   * @param hashes - The array or string containing the address(es) of the account.
-   * @param filterOptions - The filter options associated with the request.
-   * @returns The balance data of the account(s).
+   * @param {Array} hashes - The array or string containing the address(es) of the account.
+   * @param {object} [filterOptions] - The filter options associated with the request. See [docs](https://docs.amberdata.io/reference#account-balances) for more details.
+   * @returns {Promise<object>} The balance data of the account(s).
    * @example
+   * const balances = await web3data.address.getMultipleBalances(['0x06012c8cf97bead5deae237070f9587f8e7a266d', '0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be'])
```

Comment:
@taylorjdawson I thought we discussed supporting multiple addresses in the "getBalance" method. This is less confusing and simplifies logic for developers

Reply:
Similar to [above](https://github.com/web3data/web3data-js/pull/61#discussion_r342744009). But I think we just need a plan in place to simplify account balance retrieval across the board.